web/src/containers/admin/AdminDashboard.js

```diff
@@ -0,0 +1,60 @@
+import React, { Fragment } from 'react';
+import { Link } from 'react-router-dom';
+
+import Icon, { faUserCog, faUserPlus } from '../../components/Icons';
+import Footer from '../../components/Footer';
+import Header from '../../components/Header';
+
+const AdminDashboard = () => {
+  return (
+    <Fragment>
+      <Header />
+      <div className="site-body ds-l-container">
+        <div className="ds-l-row">
+          <div className="ds-l-col--8 ds-u-margin-x--auto ds-u-padding-y--6">
+            <h1 className="ds-h1">
+              Administrator actions
+            </h1>
+            <ul className="ds-c-list--bare ds-l-row">
+              <li className="ds-l-col--12 ds-l-md-col--6 ds-u-padding-left--0">
+                <h2 className="ds-h2">
+                  <div className="user-icon">
+                    <Icon icon={faUserCog} />
+                  </div>
+                  <Link
+                    to={{ pathname: '/admin/create-account' }}
+                    className="ds-u-display--block"
+                  >
+                      Create new user account
+                    </Link>
+                </h2>
+                <p>
+                  Make new accounts for federal analysts and state filers
+                </p>
+              </li>
+              <li className="ds-l-col--12 ds-l-md-col--6 ds-u-padding-right--0">
```

Comment:
I'm okay with styling this in utility classes, though, because it's unlikely we'll need to use these styles anywhere else on the site.